.github/workflows/release.yaml

```diff
@@ -78,18 +79,19 @@ jobs:
               autonomy init --reset --author $AUTHOR --ipfs --remote
               autonomy fetch $AUTHOR/$SERVICE --service --local || exit 1
               cd $SERVICE || exit 1
-              autonomy build-image || exit 1
-              autonomy build-image --version $VERSION || exit 1
-
+              autonomy build-image --image-author $DOCKER_USER || exit 1
+              docker tag $DOCKER_USER/oar-$AGENT:$DEFAULT_IMAGE_TAG $DOCKER_USER/oar-$AGENT:$VERSION || exit 1
+              docker tag $DOCKER_USER/oar-$AGENT:$DEFAULT_IMAGE_TAG $DOCKER_USER/oar-$AGENT:latest || exit 1
 
       - name: Docker login
         run: |
             echo  $DOCKER_PASSWORD | docker login -u $DOCKER_USER --password-stdin
       - name: Docker Push
         run: |
           source env.sh
-          echo "Pushing $DOCKER_USER/$SERVICE:$VERSION"
-          echo "Pushing $DOCKER_USER/$SERVICE:$DEFAULT_IMAGE_TAG"
-          docker push $DOCKER_USER/oar-$SERVICE:$VERSION
-          docker push $DOCKER_USER/oar-$SERVICE:$DEFAULT_IMAGE_TAG
-
+          for TAG in $VERSION $DEFAULT_IMAGE_TAG latest
+          do
+              IMAGE_NAME="$DOCKER_USER/oar-$AGENT:$TAG"
+              echo "Pushing $IMAGE_NAME"
+              docker push $IMAGE_NAME
+          done
```

Comment:
Are you sure this works without the ;

See above comment

Reply:
Yeah that looks good, would recommend you make your test bash.sh script much more comprehensive to ensure it functions correctly with the expected set env vars. 
I.e. doe you get the images tagged you expect that align with the output of open autonomy deploy build

Reply:
As far as I can tell from testing it, it works as expected.

Reply:
we have multiple times had issues with this.
Can you please execute the code and provide proof of this, so we dont waste more time.

Reply:
It is however of course your repo, but i will not approve without this, feel free to find another reviewer :)

Reply:
Lgtm thanks friend